tests/integration.rs

```diff
@@ -19,6 +21,8 @@ fn compiler_flag(phase: CompilerPhase) -> &'static str {
     match phase {
         CompilerPhase::Lexer => "--lextest",
         CompilerPhase::Parser => "--parsetest",
+        CompilerPhase::AstStructurePrint => "--print-ast",
+        CompilerPhase::AstPrettyPrint => "--debug-dumpast",
```

Comment:
`--debug-dumpast` was not meant to be a stable interface... the tests leak the implementation detail of AST node type names, etc.

Reply:
ok, ill remove it.